### PR TITLE
add retry for reporting twistcli results

### PR DESCRIPTION
**Pull request by YaaraVerner**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: checkov/common/bridgecrew/vulnerability_scanning/integrations/twistcli.py [hunk 1]
```diff
@@ -50,12 +51,18 @@ def report_results(self, twistcli_scan_result: Dict[str, Any], file_path: Path,
...
+
+        for i in range(2):
+            response = requests.request("POST", f"{self.vulnerabilities_base_url}/report", headers=headers, json=payload)
+            try:
+                response.raise_for_status()
+                break
+            except requests.exceptions.HTTPError as ex:
+                logging.error(f"HTTP error on request {self.vulnerabilities_base_url}/report, payload:\n{payload}")
+                if ex.response.status_code >= 500 and i != 1:
+                    time.sleep(2000)
```
Comment (gruebel) [inline checkov/common/bridgecrew/vulnerability_scanning/integrations/twistcli.py:63]: isn't it 2000 seconds? did you want to wait 2 seconds or 20?

**Review by gruebel — COMMENTED** [review]
I hope 2 seconds

File: checkov/common/bridgecrew/vulnerability_scanning/integrations/twistcli.py (hunk 1 above)
Comment (YaaraVerner) [reply checkov/common/bridgecrew/vulnerability_scanning/integrations/twistcli.py]: my bad 😅

**Review by gruebel — APPROVED** [review]
🚀

**Pull request merged**